packages/core/admin/ee/server/register.js

```diff
@@ -8,8 +8,8 @@ module.exports = async ({ strapi }) => {
   if (features.isEnabled('audit-logs')) {
     const auditLogsService = createAuditLogsService(strapi);
     strapi.container.register('audit-logs', auditLogsService);
-    auditLogsService.register();
+    await auditLogsService.register();
   }
 
-  await executeCERegister({ strapi });
```

Comment:
It's not an async function so it didn't do anything, but it's probably not our business to change that, I'll restore it

Reply:
Also you may have noticed that I made the register async too, it's not currently needed, but I was thinking it could be useful in the future for other providers, for example if you need to make API calls to AWS